content/en/code_analysis/faq/_index.md

```diff
@@ -17,17 +15,19 @@ further_reading:
 
 ## Overview
 
-Datadog Code Analysis helps you identify and address issues in your code with Static Analysis (SAST) and Software Composition Analysis (SCA). 
+[Datadog Code Analysis][10] helps you identify and address issues in your code with Static Analysis (SAST) and Software Composition Analysis (SCA). 
 
-Static Analysis reviews your code for maintainability, bugs, performance, and security issues, offering suggestions to fix them before they reach production. Software Composition Analysis examines open source libraries in your repositories to detect known vulnerabilities. 
+This page provides a non-exhaustive list of frequently asked questions about Code Analysis.
 
-### Datadog-hosted scans
+## Common questions
+
+### What are Datadog-hosted scans?
 
 With Datadog-hosted scans, your code is scanned within Datadog's infrastructure as opposed to within your CI pipeline. Datadog clones your code, runs the static analyzer to perform Static Analysis and/or Software Composition Analysis, and uploads the results.
```

Comment:
Comment in #documentation from Daniel B, leaving this here in case we want to add it in.

```suggestion
With Datadog-hosted scans, your code is scanned within Datadog's infrastructure as opposed to within your CI pipeline. Datadog reads your code, runs the static analyzer to perform Static Analysis and/or Software Composition Analysis, and uploads the results.
```